proto/wippersnapper/ds18x20/v1/ds18x20.proto

```diff
@@ -13,9 +14,10 @@ import "wippersnapper/i2c/v1/i2c.proto";
 * NOTE: This API only supports ONE DS18X20 device PER OneWire bus.
 */
 message Ds18x20InitRequest {
-  int32  onewire_pin        = 1; /** The desired pin to use as a OneWire bus. */
-  uint32 sensor_period      = 2; /** The desired amount of time between sensor reads. */
-  int32  sensor_resolution  = 3; /** The desired sensor resolution (9, 10, 11, or 12 bits). */
+  int32  onewire_pin                                        = 1; /** The desired pin to use as a OneWire bus. */
+  uint32 sensor_period                                      = 2; /** The desired amount of time between sensor reads. */
```

Comment:
`period` is also a field inside `I2CDeviceSensorProperties`, so i think it's redundant here now?

Reply:
Resolved in next commit (likely will fail due to field # changes)